cs/quickstart/authentication.texy

```diff
@@ -161,7 +159,7 @@ Poslední a **nejdůležitější věc** je **zabezpečit callbacky formulářů
 Tomu můžeme zabránit přidáním jednoduchého *if*, který přidáme na začátek metody `postFormSucceeded`:
 
 /--php
-	public function postFormSucceeded($form)
+	public function postFormSucceeded(Nette\Application\UI\Form $form)
```

Comment:
Tam je `use Nette\Application\UI\Form;`, takže stačí `Form`

Reply:
Ja to tam prave nikde nenasel, tak jsem to tam nechal. Je to konzistentni s ostatnimi casti, fakt bys to vyhodil? 

Ono to pak vypada takto:

`postFormSucceeded(Nette\Application\UI\Form $form, Nette\Utils\ArrayHash $values)`

vs

`postFormSucceeded(Form $form, Nette\Utils\ArrayHash $values)`

Reply:
No jelikož je tam ` $form = new Form;`, připadá mi konzistentní právě `postFormSucceeded(Form $form`